code-challenges.js

```diff
@@ -7,43 +7,102 @@
 // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in any two strings and returns the string with more characters. Use the two sets of test variables provided.
 
 // Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
+// const fruit1 = "apple"
+// const fruit2 = "banana"
 
-// Set two:
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
+// // Set two:
+// const fruit3 = "cherry"
+// const fruit4 = "kiwi"
 
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-const temp1 = 42
-const temp2 = 350
-const temp3 = 212
+// The goal is to create a function that takes the numbers listed and determine whether they are at, below, or above boiling point
 
+// If temp is > 212 return below boiling point
+
+// If temp is < 212 return above boiling point
+
+// If temp is === 212 return at boiling point
+
+
+// const temp1 = 42
+// const temp2 = 350
+// const temp3 = 212
+
+
+        // const boilingPoint = (number) => {
+        // if(number < 212) {
+        //     return `${number} is below boiling point`
+        // } else if (number > 212) {
+        //     return `${number} is above boiling point`
+        // } else if (number === 212) {
+        //     return `${number} is at boiling point`
+    //     }
+    // }
+        // console.log(boilingPoint(350))
+        // console.log(boilingPoint(42))
+        // console.log(boilingPoint(212))
```

Comment:
Great work Sean! Your code communicates clearly what is going on and the function is entirely dynamic!